es-models/ssm_occurrence_centric/ssm_occurrence_centric.mapping.yaml

```diff
@@ -1,5 +1,10 @@
 _all:
   enabled: false
+_size:
+  enabled: true
+_source:
+  excludes:
+  - __comment__
```

Comment:
similar as the comment in `ssm_centric` mapping, when build index for DAVE CA, pruning needs to be enabled, specific: `excludes: case.observation.*`.